docs/guide/cache/fastly.md

```diff
@@ -0,0 +1,467 @@
+
+### Get the details of installed snippets
+
+You can get more information, like priority, which subroutine it is attached to (`vcl_recv`, `vcl_fetch` etc. ) and the code itself using the `--verbose` option when using the `vcl snippet list` command:
+
+```
+$ fastly vcl snippet list --version=active -v
+Fastly API token provided via FASTLY_API_TOKEN
+Fastly API endpoint: https://api.fastly.com
+Service ID (via FASTLY_SERVICE_ID): [....]
+
+Service Version: 8
+
+Name: Re-Enable shielding on restart
+ID: 1iJWIfsPLNGxcphsjggq
+Priority: 100
+Dynamic: false
+Type: recv
+Content:
+// This code should be added a snipped in your config:
+//  Name : Re-Enable shielding on restart
+//  Priority: 100
+//  Type: recv
+//
+// Fastly CLI :
+// - fastly vcl snippet create --name="Re-Enable shielding on restart" --version=active --autoclone --priority 100 --type recv --content=vendor/ezsystems/ezplatform-http-cache-fastly/fastly/snipped_re_enable_shielding.vcl
+// - fastly service-version activate --version=latest
+
+
+set var.fastly_req_do_shield = (req.restarts <= 2);
+
+# set var.fastly_req_do_shield = (req.restarts > 0 && req.http.accept == "application/vnd.fos.user-context-hash");
+set req.http.X-Snippet-Loaded = "v1";
+
+
+Created at: 2022-06-23 10:55:34 +0000 UTC
+Updated at: 2022-06-23 12:24:48 +0000 UTC
+
+```
+
+You can also get the same details for a particular snippet using the `vcl snippet describe` command :
+
+```
+fastly vcl snippet describe --version=active "--name=Re-Enable shielding on restart"
+```
+
+### Create a snippet
+
+```
+fastly vcl snippet create --name="Re-Enable shielding on restart" --version=active --autoclone --priority 100 --type recv --content=vendor/ezsystems/ezplatform-http-cache-fastly/fastly/snippet_re_enable_shielding.vcl
+fastly service-version activate --version=latest
+```
+
+### Delete a snippet
+
+```
+fastly vcl snippet delete --name="Re-Enable shielding on restart" --version=active --autoclone
+fastly service-version activate --version=latest
+```
+
+### Update an existing snippet
+
+```
+fastly vcl snippet update --name="Re-Enable shielding on restart" --version=active --autoclone --priority 100 --type recv --content=vendor/ezsystems/ezplatform-http-cache-fastly/fastly/snippet_re_enable_shielding.vcl
+fastly service-version activate --version=latest
+```
+
+### Get the diff between two versions
+
+Using the Fastly web interface you can easily see the diff between two different versions. Unfortunately, this is not supported by the Fastly CLI. However, it is possible with the help of the Fastly API and GNU diff.
+
+Use the Fastly API to download the generated VCL. This generated VCL is the vcl config that Fastly generates based on all the configuration settings (so it includes all custom .vcls, snippets, origin config).
```

Comment:
```suggestion
Use the Fastly API to download the generated VCL file. It includes the VCL configuration that Fastly generates based on all the configuration settings (from all custom `.vcl` files, snippets, and origin configuration).
```

If "origin" is used in a meaning of "original", please replace here and in other occurrences.